pyproject.toml

```diff
@@ -15,6 +15,7 @@ Sphinx = ">=1.3"
 sphinx_rtd_theme = ">=0.3.1"
 behave = ">=1.2.6"
 recommonmark = ">=0.4.0"
+pytest-bdd = {git = "https://github.com/rbcasperson/pytest-bdd.git", rev = "scenario-descriptions", optional = true}
```

Comment:
Huh, I hope this reference to a personal repo is something that will evolve into a JGT repo at some point?
(tiny github PR review UI syndrome might be at work here too)

Reply:
I am still concerned about this 'personal repo' being used here...

Reply:
So I've looked a bit into this. `poetry` does support _most_ of the more advanced ways you can specify a package, but doesn't appear to properly handle parsing/passing a PR refspec down to `pip`/`git`. Our options, as best I can determine, are:
* sit on this addition until [Ryan's PR](https://github.com/pytest-dev/pytest-bdd/pull/312) merges (it's been awaiting review since Dec 3)
* rework the pytest-bdd parser class to strip out scenario descriptions for now and just add support for scenario descriptions back later when the PR merges
* run with the personal branch from which the PR is sourced until that PR merges (then come back and return our ref to the "official" `pytest-bdd` source)

Working on a team that could benefit today from the full range of what this work currently offers, I'm obviously biased toward option (3) there.

Reply:
Isn't there maybe a 4th option?
What about using the stock pytest-bdd parser as is, in which case the extra functionality won't come out of the stock parser, but then anyone who wanted to layer in Ryan's PR after the install would get it and the sphinx-gherkindoc part would be ready?
(Sorry, this is a dash-off quick half-baked thought as I didn't have any more time today at lunch)